src/RcppExports.cpp

```diff
@@ -10,24 +11,21 @@ Rcpp::Rostream<true>&  Rcpp::Rcout = Rcpp::Rcpp_cout_get();
 Rcpp::Rostream<false>& Rcpp::Rcerr = Rcpp::Rcpp_cerr_get();
 #endif
 
-// adder
-int adder(int a, int b);
-RcppExport SEXP _frogger_adder(SEXP aSEXP, SEXP bSEXP) {
+// run_base_model
+Rcpp::List run_base_model(const Rcpp::List demp, SEXP sim_years);
+RcppExport SEXP _frogger_run_base_model(SEXP dempSEXP, SEXP sim_yearsSEXP) {
 BEGIN_RCPP
     Rcpp::RObject rcpp_result_gen;
     Rcpp::RNGScope rcpp_rngScope_gen;
-    Rcpp::traits::input_parameter< int >::type a(aSEXP);
-    Rcpp::traits::input_parameter< int >::type b(bSEXP);
-    rcpp_result_gen = Rcpp::wrap(adder(a, b));
+    Rcpp::traits::input_parameter< const Rcpp::List >::type demp(dempSEXP);
```

Comment:
is the whitespace inside angle brackets canonical styling? intellij does this too but I thought it was a bug

Reply:
That was a pretty common style in pre-C++11 style because it helped disambiguate `>>` from `> >`, but is discouraged now. That file is autogenerated file, but in stuff we write no spaces is better